rules/quality.bzl

```diff
@@ -336,3 +336,56 @@ html_coverage_report = rule(
     },
     executable = True,
 )
+
+def _modid_check_aspect_impl(target, ctx):
+    """
+    Verify that a binary (ELF file) does not contain conflicting module IDs
+    using opentitantool. The result of this aspect is put is in modid_check
```

Comment:
Nit: Stray "is"

Reply:
Same here: part of a commit from a #19062, I will address this in a follow up.